### DISCUSSION TITLE
KEDA Chart v2.13.0

### DISCUSSION THREAD

**Discussion by tomkerkhove** [kedacore/charts#594] [Announcements]
Chart: v2.13.0 | App: v2.13.0

## TL;DR

```shell
helm repo add kedacore https://kedacore.github.io/charts
helm repo update
helm install keda kedacore/keda --version 2.13.0 -n keda --create-namespace
```

### Features

- Support for KEDA v2.13.0 ([release notes](https://github.com/kedacore/keda/releases/tag/v2.13.0))
- Support profiling for KEDA components (#549)
- Support for disabling compression (#554)
- Support for  CRD-specific annotations (#584)
- Support for Cilium network policies (#558)
- Support for TLS configuration for ServiceMonitor (#591)
- Allow image registry override for all keda components (#557)
- Introduce RBAC rule to allow access to LimitRange (#588)
 
### Fixes / Changes

- Added configuration for validation webhook (#553)
- Add missing port in KEDA operator (#552)
- Fix the service port name for webhook to avoid breaking istio (https://github.com/kedacore/charts/pull/551)
- Show only logs with a severity level of ERROR or higher in the stderr (#506)

### Breaking Changes

None.

### Removal

None.

## New Contributors
* @loispostula made their first contribution in https://github.com/kedacore/charts/pull/548
* @congzhegao made their first contribution in https://github.com/kedacore/charts/pull/551
* @Adarsh-verma-14 made their first contribution in https://github.com/kedacore/charts/pull/506
* @yuvalweber made their first contribution in https://github.com/kedacore/charts/pull/549
* @radekfojtik made their first contribution in https://github.com/kedacore/charts/pull/553
* @QuentinBisson made their first contribution in https://github.com/kedacore/charts/pull/557
* @eumel8 made their first contribution in https://github.com/kedacore/charts/pull/561
* @aballman made their first contribution in https://github.com/kedacore/charts/pull/567
* @Bhargav-InfraCloud made their first contribution in https://github.com/kedacore/charts/pull/588
* @ferndem made their first contribution in https://github.com/kedacore/charts/pull/590
* @mplzik made their first contribution in https://github.com/kedacore/charts/pull/555
* @choisungwook made their first contribution in https://github.com/kedacore/charts/pull/556
* @awalford16 made their first contribution in https://github.com/kedacore/charts/pull/584
* @wozniakjan made their first contribution in https://github.com/kedacore/charts/pull/593

**Full Changelog**: https://github.com/kedacore/charts/compare/v2.12.0...v2.13.0

<hr /><em>This discussion was created from the release <a href='https://github.com/kedacore/charts/releases/tag/v2.13.0'>KEDA Chart v2.13.0</a>.</em>
